app.js

```diff
@@ -1,148 +1,106 @@
-  );
-
-  if (selectedValue && selectedValue.value === "b") {
-    alert("¡Respuesta correcta!");
-    score++;
-    document.getElementById("question-3-card").style.display = "none";
-    document.getElementById("question-4-card").style.display = "block";
-  } else {
-    alert("Respuesta incorrecta. La respuesta correcta es Esquimales.");
-    document.getElementById("question-3-card").style.display = "none";
-    document.getElementById("question-4-card").style.display = "block";
-  }
   printScore();
 }
 
-//CUARTO formulario
-function handleQuestion4Submit(event) {
-  event.preventDefault();
-
-  const selectedValue = document.querySelector(
-    'input[name="questionFour"]:checked'
-  );
-
-  if (selectedValue && selectedValue.value === "b") {
-    alert("¡Respuesta correcta!");
-    score++;
-    document.getElementById("question-4-card").style.display = "none";
-    document.getElementById("question-5-card").style.display = "block";
-  } else {
-    alert("Respuesta incorrecta. La respuesta correcta es un dinosaurio.");
-    document.getElementById("question-4-card").style.display = "none";
-    document.getElementById("question-5-card").style.display = "block";
-  }
-  printScore();
-}
-
-// QUINTO formulario
-function handleQuestion5Submit(event) {
-  event.preventDefault();
-
-  const selectedValue = document.querySelector(
-    'input[name="questionFive"]:checked'
-  );
-
-  if (selectedValue && selectedValue.value === "c") {
-    alert("¡Respuesta correcta!");
-    score++;
-    document.getElementById("question-5-card").style.display = "none";
-  } else {
-    alert("Respuesta incorrecta. La respuesta correcta es Bola de Nieve.");
-    document.getElementById("question-5-card").style.display = "none";
-  }
-  printScore();
-  document.getElementById("result-container").style.display = "block";
-}
-
-// Mostrar el resultado general
+// Show general result
 function printScore() {
   document.getElementById(
     "score"
   ).textContent = `Puntaje: ${score} de ${totalQuestions}`;
 }
 
-// Obtener el parámetro "nombre" de la URL
+// Get param "name" from URL
 const urlParams = new URLSearchParams(window.location.search);
 const nombre = urlParams.get("playerName");
 
-// Mostrar el saludo en la página
+// Show greeting in the website
 const messageDiv = document.getElementById("player-name-hello");
 messageDiv.innerText = ` Hola, ${nombre}.`;
 
-// Agregar event listeners a los formularios
-document
-  .getElementById("question1")
-  .addEventListener("submit", handleQuestion1Submit);
-
-document
-  .getElementById("question2")
-  .addEventListener("submit", handleQuestion2Submit);
-
-document
-  .getElementById("question3")
-  .addEventListener("submit", handleQuestion3Submit);
-
-document
-  .getElementById("question4")
-  .addEventListener("submit", handleQuestion4Submit);
-
-document
-  .getElementById("question5")
-  .addEventListener("submit", handleQuestion5Submit);
+// Add event listeners to the forms
```

Comment:
This is good. It can be better if in a next iteration you create a json with the answers instead. ie.
```
const answers = [{
  correctAnswer: 'b',
  correctMessage: 'bien hecho!',
  ...
}, ...]

if(i === 1) {
 const {
  correctAnswer,
  correctMessage,
} = answers[i]
}

```

Reply:
read more about destructuring objects